FFXCutsceneRemover/Components/MemoryWatchers.cs

```diff
@@ -119,7 +138,14 @@ public void Initialize(Process process)
                     PartyMembers,
                     Sandragoras,
                     HpEnemyA,
-                    GuadoCount
+                    GuadoCount,
+                    EnableAuron,
+                    LucaFlag,
+                    LucaFlag2,
+                    MiihenFlag1,
+                    MiihenFlag2,
+                    MiihenFlag3,
+                    MiihenFlag4
```

Comment:
Is it possible to provide more useful names for these guys?

Like for example one of the mi'ihen flags skips the convo with Shelinda, right? Can we call it ShelidaMiihenFlag or something like this?